python/share/jupyter/nbconvert/templates/flex/index.html.j2

```diff
@@ -23,11 +23,6 @@
     {% block html_head_css -%}
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet"/>
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css" integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
-    {% if dev_mode -%}
-    <link rel="stylesheet" href="http://localhost:8866/voila/static/dist/jupyter-flex-embed.css" />
-    {%- else -%}
-    {{ resources.include_css("static/dist/jupyter-flex-embed.css") }}
-    {%- endif %}
 	{% block flex_custom_css -%}
```

Comment:
We need to but the custom css below the `jupyter-flex-embed.css` so custom css can overwrite our defaults.

Reply:
Ah yes, I think we might need to duplicate the custom style block as well, so users can override the loading style as well as the main dashboard